blender-mcp.md

```
@@ -0,0 +1,154 @@
+# Usage with official Blender MCP
+
+## Prerequisites
+
+In a folder of your choice (e.g. `C:\Users\username\src\blender\`):
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)